vispy/scene/cameras/base_camera.py

```diff
@@ -108,6 +109,19 @@ def depth_value(self, value):
             raise ValueError('depth value must be positive')
         self._depth_value = value
 
+    @property
```

Comment:
Can you combine this with `zoom_factor` in a way that makes them appear related? (would be fine to rename it if you choose)